content/docs/use-cases/secure-data-caching-hub.md

```diff
@@ -0,0 +1,78 @@
+# Secure Data and Model Caching Hub
```

Comment:
let's not make it about models I think

this layer is important primarily to optimize datasets usage

Reply:
minor: `Hub` - do we need this new term? I'm fine to keep it. just doesn't feel like it's needed

Reply:
by using Secure exclusively we are overemphasizing it too much now ... it become specifically about "security" now

Reply:
`Caching Hub` can be replaced with `Cache` I think?

Reply:
> let's not make it about models

OK, removing "and Models".

> Hub - do we need this new term?
Caching Hub can be replaced with Cache

Could be "Layer", or nothing. Layer could refer to the infrastructural aspect of this patter but it's not clear until you actually read the contents. What I like abut Hub is that a) come people may get that it's infrastructure-related **and** it implies multiple parties (people or projects in this case) can employ the same data caching mechanism.

> by using Secure exclusively we are overemphasizing it too much now

Confused. So don't include "Secure"?

Reply:
> by using Secure exclusively

Ah I see so you mean because it was "Secure and Fast" before. Well since "and Models" is gone now, we can go back to "Fast and Secure" but fast has some misleading connotations (e.g. improving network throughput)... 🤔